.github/workflows/release.yml

```
@@ -1,26 +1,18 @@
-name: Build
+name: Release
 
 on:
-  pull_request:
-    branches:
-      - main
-
   push:
     branches:
       - main
       - beta
 
 jobs:
   release:
-    name: Release
+    name: build-lint-test
```

I guess this was fine as it was?

```suggestion
    name: release
```